source/docs/04-cmn.xml

```diff
@@ -1104,10 +1104,33 @@
                   <p>
                      <figure>
                         <head/>
-                        <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/cmn/cmn-sample158.txt" parse="text"/></egXML>
+                        <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/cmn/cmn-ossia_layer.txt" parse="text"/></egXML>
                      </figure>
                   </p>
                </div>
+               <div xml:id="cmnCue" type="div3">
+                  <head>Cue</head>
+                  <p>Cue notes are smaller notes that usually occur in an orchestral or vocal part and are not to be played or sung by the corresponding musician, but by other instruments or singers. Cue notes are used for orientation, <abbr>i.e.</abbr>, to follow the music during longer pauses and to find the correct re-entry point more easily. In MEI the <att>cue</att> attribute is available to indicate such notes.</p>
+                  <p>
+                     <figure>
+                        <head/>
+                        <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/cmn/cmn-cue_note.txt" parse="text"/></egXML>
+                     </figure>
+                  </p>
+                  <p>Most often cue notes occur in a group rather than one by one. This is because usually a whole layer of another part is inserted as a cue. Therefore, a complete <gi scheme="MEI">layer</gi> can also be marked as cue.</p>
+                  <p>
+                     <figure>
+                        <head>Cue notes in the flute part of Brahms’ <hi rend="italic">Ein deutsches Requiem</hi>.</head>
+                        <graphic url="../images/modules/cmn/Brahms-flutePart.png"/>
+                     </figure>
+                     <figure>
+                        <head/>
+                        <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve" valid="true"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/cmn/cmn-cue_layer.txt" parse="text"/></egXML>
+                     </figure>
+                  </p>
+                  <p>If the voice from which the cue notes originate is also encoded, they should refer to their sounding counterpart with the <att>corresp</att> attribute.</p>
```

Comment:
Would it be possible to include another encoding example here for this linking mechanism with `corresp`. This would be very helpful for users consulting this passage.

Reply:
As you suggested this should probably go into [chapter 13](https://music-encoding.org/guidelines/dev/content/analysisharm.html#analysisDescribingRelationships) where nothing can be found about `@corresp`at all. Opened #915.

Reply:
`@corresp` itself and all other attributes of `att.linking` are momentarily explained in chapter 10 (Analytical markup). If that's a good place for them, we should discuss over there in #915, right. 

For the passage here, I am not asking to explain `corresp` in general (what would be too much out of scope of this PR), just to add a minimal encoding example how you would target the "sounding counterpart" from the cued layer. (My initial thought was to link to the corresp section from here, but since there is nothing really helpful for the use case here, it seems easier for now to include a small example here which exemplifies that usage of cued layers.) What do you think?